stubs/babel/babel/messages/catalog.pyi

```diff
@@ -1,108 +1,126 @@
-from typing import Any
+import datetime
+from collections import OrderedDict
+from collections.abc import Generator, Iterable, Iterator
+from typing_extensions import TypeAlias
+
+from babel.core import Locale
+
+__all__ = ["Message", "Catalog", "TranslationError"]
+
+_MessageID: TypeAlias = str | tuple[str, ...] | list[str]
```

Comment:
Why isn't this just `str | tuple[str, str]`? According to the docstrings, this is either a message string or a (singular, plural) pair.

Reply:
It's possible that more than one plural message could exist (depending on the language), and the code treats lists/tuples interchangeably in all the checks.